Add Source Identifier for Feather compatibility.

yodaluca23 opened the pull request:
[Feather](https://github.com/khcrysalis/Feather) relies on the source having a root "identifier" key. #126 broke this. Re-adding ```"identifier": "com.stik.sj"``` to the root fixes this.

CelloSerenity commented:
I wondered why that was there when I rewrote the json. Glad to hear it was necessary and fixed!

yodaluca23 commented:
> I wondered why that was there when I rewrote the json. Glad to hear it was necessary and fixed!


I think the AltStore spec also used to mandate it.

CelloSerenity commented:
Probably, it just wasn't in the current docs for Alt or Side.
